portable/GCC/ARM_CM4F/port.c

```diff
@@ -304,7 +304,7 @@ BaseType_t xPortStartScheduler( void )
 
     #if ( configASSERT_DEFINED == 1 )
     {
-        volatile uint32_t ulOriginalPriority;
+        volatile uint8_t ulOriginalPriority;
```

Comment:
This seems okay but it needs to be done across multiple ports. Do you want to do that too or would you prefer us to do that?

Reply:
Yes, I have created #658 . Thanks.